pkg/reconciler/v1alpha1/autoscaling/kpa/kpa.go

```diff
@@ -21,6 +21,9 @@ import (
 	"fmt"
 	"reflect"
 
+	"fortio.org/fortio/log"
```

Comment:
I guess that's not supposed to be in here?

Reply:
+1

Reply:
Not really, autoimport :)